libnetwork/iptables/firewalld_test.go

```diff
@@ -18,9 +19,13 @@ func skipIfNoFirewalld(t *testing.T) {
 	}
 	defer conn.Close()
 
-	var zone string
-	err = conn.Object(dbusInterface, dbusPath).Call(dbusInterface+".getDefaultZone", 0).Store(&zone)
-	if err != nil {
+	currentState, err := connection.sysObj.GetProperty(dbusInterface + ".state")
+	var running bool
+	if err == nil {
+		running, _ = regexp.MatchString(".*RUNNING.*", currentState.Value().(string))
+	}
```

Comment:
Use string comparisons instead of regexp; don't panic if `Value()` is not a string; indent error flow. All the review comments from firewalld.go are also applicable here.